map/gps_track_storage.cpp

```diff
@@ -66,6 +69,31 @@ void Unpack(char const * p, location::GpsInfo & info)
   info.m_source = static_cast<location::TLocationSource>(source);
 }
 
+inline size_t GetItemOffset(size_t itemIndex)
+{
+  return sizeof(uint32_t) /* header */ + itemIndex * kPointSize;
```

Comment:
size_t const kHeaderSize = sizeof(uint32_t);
?
After that you can get rid of these comments (and duplication).